### PR TITLE
Fix meryl output label

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

Fix PR fixes the label of the pat.meryl hap-mer db.

### CODE REVIEW

File: tools/meryl/meryl.xml [hunk 1]
```diff
@@ -220,7 +220,7 @@
         <data name="read_db_hist" format="tabular" from_work_dir="read-db.hist" label="${tool.name} on ${on_string}: read-db histogram">
             <filter>operation_type["command_type"] == "histogram-kmers" or operation_type["command_type"] == "trio-mode"</filter>
         </data>
-        <data name="pat_db" format="meryldb" from_work_dir="pat.meryldb" label="${tool.name} on ${on_string}: read-db.meryl">
+        <data name="pat_db" format="meryldb" from_work_dir="pat.meryldb" label="${tool.name} on ${on_string}: pat.meryl">
```
Comment (bgruening) [inline tools/meryl/meryl.xml:223]: meryldb as above?

Comment (gallardoalba) [reply tools/meryl/meryl.xml:223]: yes, only the label is wrong

Comment (bgruening) [reply tools/meryl/meryl.xml:223]: you mix `meryl` and `meryldb` should that be consistent?

Comment (gallardoalba) [reply tools/meryl/meryl.xml:223]: I can replace the extensions of the labels, probably is less confusing in that way.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @gallardoalba!

**Pull request merged**